### PR TITLE
Fixing 2.6.0 blog post

### PR DESCRIPTION
**Pull request by pabloem**
r: @ThatRfernand

### CODE REVIEW

**Comment by ThatRfernand** [comment]
Tested with a local instance of Jekyll, which I recommend you setup when you work with beam-site. Sent you a fix for the layout issues:

- There was a bug introduced in PR #481 -- Jekyll does not do well with Markdown comments in code[1].
- You needed the "More" tag in the original text so it renders correctly.

[1] https://stackoverflow.com/questions/22106698/jekyll-markdown-html-comment-tag-converted-to-literals

**Comment by ThatRfernand** [comment]
R: @melap

File: src/_posts/2018-08-10-beam-2.6.0.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+available for runner integration
+* A Universal Local Runner has been added to Beam. This runner
+runs in a single machine using portability, and containerized
+SDK harnesses.
+* Flink Runner now accepts jobs using the Job API
+
+### IOs
+
+* Bounded SplittableDoFn support is now available in all
+runners (Note SDF is the new I/O connector API)
```
Comment (melap) [inline src/_posts/2018-08-10-beam-2.6.0.md]: I'd recommend removing "Note" or changing it to "Note that"

File: src/_posts/2018-08-10-beam-2.6.0.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+* HBaseIO is the first I/O supporting Bounded SDF (using
+  readAll)
+* Miscellaneous Fixes
+
+### SDKs
+
+* Faster Python Avro IO
+* Python Avro IO has a use_fastavro option that will use
+fastavro instead of apache/avro, for a
+[3-6xspeedup!](https://gist.github.com/ryan-williams/ede5ae61605e7ba6aa655071858ef52b)
```
Comment (melap) [inline src/_posts/2018-08-10-beam-2.6.0.md]: missing space between 3-6x and speedup

File: src/_posts/2018-08-10-beam-2.6.0.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### Other
+
+* Updates to various dependency versions
+* Improvements on stability, performance, and documentation
```
Comment (melap) [inline src/_posts/2018-08-10-beam-2.6.0.md]: on --> to

**Comment by melap** [comment]
Staged: http://apache-beam-website-pull-requests.storage.googleapis.com/536/blog/index.html

File: src/_posts/2018-08-10-beam-2.6.0.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+* Python Avro IO has a use_fastavro option that will use
+fastavro instead of apache/avro, for a
+[3-6x speedup!](https://gist.github.com/ryan-williams/ede5ae61605e7ba6aa655071858ef52b)
+
+### Other
+
+* Updates to various dependency versions
+* Improvements to stability, performance, and documentation
+
+# List of Contributors
```
Comment (melap) [inline src/_posts/2018-08-10-beam-2.6.0.md]: this should be down a level in heading with removal of the title - ##

File: src/_posts/2018-08-10-beam-2.6.0.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+        - pabloem
+        - rfernand
+
+---
+
+We are glad to present the new 2.6.0 release of Beam.
+This release includes multiple fixes and new functionalities.
+Specifically, we spent a significant amount of time trying to
+automate the release, and fix the continuous integration. We
+also have significant new features in SQL, and portability.<!--more--> For more information, please, check the detailed
```
Comment (melap) [inline src/_posts/2018-08-10-beam-2.6.0.md]: remove comma -->  "SQL and portability"

**Comment by pabloem** [comment]
Retest this please

**Comment by pabloem** [comment]
@asfgit merge

**Comment by asfgit** [comment]
Error: PR failed in verification; check the Jenkins job for more information.

**Comment by pabloem** [comment]
@asfgit merge

**Pull request closed without merging**